src/js/components/Calendar/Calendar.js

```diff
@@ -154,6 +155,19 @@ const Calendar = forwardRef(
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
 
+    // set activeRangeBound when caller changes it, allows us to change
+    // it internally too
+    const activeRangeBounds = {
```

Comment:
This could be outside the component function, since it's a constant, perhaps line 43.